org.unitystation.StationHub.yaml

```diff
@@ -0,0 +1,71 @@
+    - install -D org.unitystation.StationHub.desktop /app/share/applications/org.unitystation.StationHub.desktop
+    - install -D org.unitystation.StationHub64.png /app/share/icons/hicolor/64x64/apps/org.unitystation.StationHub.png
+    - install -D org.unitystation.StationHub128.png /app/share/icons/hicolor/128x128/apps/org.unitystation.StationHub.png
+    - install -D org.unitystation.StationHub256.png /app/share/icons/hicolor/256x256/apps/org.unitystation.StationHub.png
+    - install -D org.unitystation.StationHub512.png /app/share/icons/hicolor/512x512/apps/org.unitystation.StationHub.png
+    - install -D org.unitystation.StationHub.metainfo.xml /app/share/metainfo/org.unitystation.StationHub.metainfo.xml
+    sources:
+      - type: file
+        path: org.unitystation.StationHub64.png
+      - type: file
+        path: org.unitystation.StationHub128.png
+      - type: file
+        path: org.unitystation.StationHub256.png
+      - type: file
+        path: org.unitystation.StationHub512.png
+      - type: file
+        path: org.unitystation.StationHub.desktop
+      - type: file
+        path: org.unitystation.StationHub.metainfo.xml
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/dotnet/install.sh
+  - name: StationHub
+    sources:
+      - type: git
+        url: https://github.com/unitystation/stationhub
+        commit: 0e6dc7ec52d6c80fd242e177aa4319b54821a501
+        branch: develop
+      - type: file
+        path: nuget.config
+      - type: file
+        url: https://download.visualstudio.microsoft.com/download/pr/5d139dff-4ca0-4e0c-a68b-0976281d5b2d/d306f725466e058842faa25bf1b2f379/dotnet-runtime-3.1.0-linux-x64.tar.gz
+        sha512: 99949807c00871d66e8ce7c25c14998e78a0ea60ba8cc42244643ed2e13aa360285df1c8d27729df3efb319f4af9163ea5626c1478a9dd4bed9d2a58e01d6343
+      - type: file
+        url: https://download.visualstudio.microsoft.com/download/pr/d731f991-8e68-4c7c-8ea0-fad5605b077a/49497b5420eecbd905158d86d738af64/dotnet-sdk-3.1.100-linux-x64.tar.gz
+        sha512: 5217ae1441089a71103694be8dd5bb3437680f00e263ad28317665d819a92338a27466e7d7a2b1f6b74367dd314128db345fa8fff6e90d0c966dea7a9a43bd21
+      - type: file
+        path: stationhub.sh
+      - sources.json
+    buildsystem: simple
+    build-commands:
+      - '. /usr/lib/sdk/dotnet/enable.sh; dotnet publish -c Release ./UnitystationLauncher/UnitystationLauncher.csproj /p:DefineConstants="FLATPAK"'
+      - cp /lib/sdk/dotnet/lib/host/fxr/3.1.6/libhostfxr.so UnitystationLauncher/bin/Release/netcoreapp3.1/
+      - cp /lib/sdk/dotnet/lib/shared/Microsoft.NETCore.App/3.1.6/libhostpolicy.so UnitystationLauncher/bin/Release/netcoreapp3.1/
+      - cp -R UnitystationLauncher/bin/Release/netcoreapp3.1/ /app/org.unitystation.StationHub
+      - install -D stationhub.sh /app/bin/stationhub.sh
+      - tar zxf dotnet-runtime-3.1.0-linux-x64.tar.gz -C /app/dotnet
+      - tar zxf dotnet-sdk-3.1.100-linux-x64.tar.gz -C /app/dotnet
```

Comment:
At first glance at least the SDK part is already there in the `dotnet` SDK extension: https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet/blob/branch/20.08/org.freedesktop.Sdk.Extension.dotnet.yaml#L40

Do you need to include another one? Should runtime be included in the SDK extension for it to be really useful? (Asking as I don't know anything about .NET)